test/functional/apps/visualize/_vertical_bar_chart.js

```diff
@@ -296,5 +296,24 @@ export default function ({ getService, getPageObjects }) {
       });
 
     });
+
+    it('should have `drop partial buckets` option', async () => {
+      await PageObjects.visualize.clickDropPartialBuckets();
+      await PageObjects.visualize.clickGo();
+
+      const expectedChartValues = [37, 202, 740, 1437, 1371, 751, 188, 31, 42, 202, 683,
+        1361, 1415, 707, 177, 27, 32, 175, 707, 1408, 1355, 726, 201, 29
+      ];
```

Comment:
I think it's better to change this test as the following:
- create a bar chart with a different timestamp than the standard one (one that contains partial buckets)
- test the expected values before applying the `drop partial bucket` option
- apply the  `drop partial bucket` option
- test and expect a smaller amount of values as expected data.

So we can test either the partial bucket option checkbox and the functionality